lib/graphql/result_cache/callback.rb

```diff
@@ -1,21 +1,32 @@
 module GraphQL
   module ResultCache
     class Callback
-      def initialize obj:, args:, ctx:, value:
+      def initialize(obj:, args:, ctx:, value:, field: nil)
         @obj = obj
         @args = args
         @ctx = ctx
         @value = value
+        @field = field
       end
 
-      def call result_hash
+      def call(result_hash)
         case @value
           when Symbol
             @obj.public_send(@value, result_hash)
           when Proc
             @value.call(result_hash, @obj, @args, @ctx)
         end
-        ::GraphQL::ResultCache.logger && ::GraphQL::ResultCache.logger.debug("GraphQL result cache callback called for #{@ctx.path.join('.')}")
+        ::GraphQL::ResultCache.logger && ::GraphQL::ResultCache.logger.debug("GraphQL result cache callback called for #{callback_caller}")
+      end
+
+      private
+
+      def callback_caller
+        ctx_path || @field.name
```

Comment:
Same would apply here: `@field.path` should be used over `@field.name`. Though it does not really matter since it only affects logging...

Reply:
💯